WORKSPACE

```diff
@@ -784,38 +786,44 @@ VIRTUALENV_PREFIX = "virtualenv-15.1.0"
 WHEEL_SRC = "https://pypi.python.org/packages/c9/1d/bd19e691fd4cfe908c76c429fe6e4436c9e83583c4414b54f6c85471954a/wheel-0.29.0.tar.gz"
 
 http_file(
-    name = 'pytest_whl',
-    url = PYTEST_WHEEL,
+    name = "pytest_whl",
+    downloaded_file_path = "pytest-3.1.3-py2.py3-none-any.whl",
+    urls = [PYTEST_WHEEL],
 )
 
 http_file(
-    name = 'py_whl',
-    url = PY_WHEEL,
+    name = "py_whl",
+    downloaded_file_path = "py-1.4.34-py2.py3-none-any.whl",
+    urls = [PY_WHEEL],
 )
 
 http_file(
     name = "wheel_src",
-    url = WHEEL_SRC,
+    downloaded_file_path = "wheel-0.29.0.tar.gz",
+    urls = [WHEEL_SRC],
 )
 
 http_file(
     name = "pex_src",
-    url = PEX_SRC,
+    downloaded_file_path = "pex-1.2.15.tar.gz",
```

Comment:
Maybe a simpler thing to do would be to define the versions at the top of the WORKSPACE file, as is already being done foe some Java dependencies.
I can make the change in this PR, though it will just add more cosmetic changes to this changeset.

Reply:
If we did that here I think it would make sense to do everywhere in the project to be consistent.